target/hexagon/idef-parser/idef-parser.lex

```diff
@@ -629,13 +633,36 @@ SIGN_ID                  s|u
 "fDEINTERLEAVE"          { return DEINTERLEAVE; }
 "fINTERLEAVE"            { return INTERLEAVE; }
 "fCARRY_FROM_ADD"        { return CARRY_FROM_ADD; }
+"size"[1248][us]"_t"     { /* Handles "size_t" variants of int types */
+                           const unsigned int bits_per_byte = 8;
+                           const unsigned int bytes = yytext[4] - '0';
+                           yylval->rvalue.bit_width = bits_per_byte*bytes;
```

Comment:
```suggestion
                           yylval->rvalue.bit_width = bits_per_byte * bytes;
```
Use `script/checkpatch.pl` to validate your patches.

Reply:
Fixed formatting issues. Thanks!